packages/lab/src/badge/Badge.css

```diff
@@ -24,44 +6,35 @@
   position: relative;
```

Comment:
I would replace this with 
```suggestion
  position: absolute;
```

Reply:
does adding "position: absolute" not position the element relative to the DOM. So in this case would appear top right of the screen ?

Reply:
No, that is `position: fixed`